incubator/hnc/config/rbac/role.yaml

```diff
@@ -13,13 +13,10 @@ rules:
   verbs:
   - create
   - delete
-  - deletecollection
   - get
-  - impersonate
   - list
   - patch
   - update
-  - use
```

Comment:
Not sure if this needs to be checked in.

Reply:
Weird. I think I removed those permissions a while ago and just forgot to regenerate the files? Maybe remove this from the commit if you can and resubmit separately?

Reply:
Done.